graphql/nearText.js

```diff
@@ -111,37 +121,59 @@ export default class GraphQLNearText {
       throw new Error("nearText filter: moveTo must be object");
     }
 
-    if (!Array.isArray(target.concepts)) {
+    if (!Array.isArray(target.concepts) && !Array.isArray(target.objects)) {
```

Comment:
should be `!target.concepts && !target.objects` to reflect error message. Array type is checked later on.